framework/src/multiapps/TransientMultiApp.C

```diff
@@ -49,15 +49,9 @@ TransientMultiApp::validParams()
 
   params.addParam<bool>("detect_steady_state",
                         false,
-                        "If true then while sub_cycling a steady state check will be "
-                        "done.  In this mode output will only be done once the "
-                        "MultiApp reaches the target time or steady state is reached");
-
-  params.addParam<Real>("steady_state_tol",
-                        1e-8,
-                        "The relative difference between the new "
-                        "solution and the old solution that will be "
-                        "considered to be at steady state");
+                        "If true, then if/while sub-cycling ('sub_cycling = true'), a steady-state "
+                        "check will be performed for each child app, allowing them to skip to the "
+                        "end of the parent time step if steady conditions are detected.");
```

Comment:
so if you set this, it makes the child apps create SteadyState cv checks.
but the other parameter is just cumbersome to make happen?

Reply:
> I don't understand what you mean, but I think this parameter can be eliminated

If we don't use it, works for me